iphone/cli/hooks/frameworks.js

```diff
@@ -579,7 +579,7 @@ class FrameworkIntegrator {
 			isa: 'PBXBuildFile',
 			fileRef: fileRefUuid,
 			fileRef_comment: frameworkPackageName,
-			settings: { ATTRIBUTES: [ 'CodeSignOnCopy' ]  }
+			settings: { ATTRIBUTES: [ 'CodeSignOnCopy', 'RemoveHeadersOnCopy' ] }
```

Comment:
I believe Xcode would automatically add `RemoveHeadersOnCopy` when you opened the generated Xcode project. I tested explicitly adding it as you've done, but as I recall it wasn't necessary, so I ripped it out. I'm fine with you adding it.

Reply:
Just tested this pull - it works fine without this setting.

Reply:
I just added it for consistency because that's the attributes we use when adding TitaniumKit as well.